pages/index.js

```diff
@@ -1,9 +1,13 @@
-import Heading from "../components/Heading";
+import NewForm from "../components/NewForm";
+
 
 export default function Home() {
+  
+
   return (
-    <main>
-      <Heading>🐠 Tropical Fish Capstone Template 🐠</Heading>
-    </main>
-  );
+    
+      <main>
+        <NewForm/>
+      </main>
+  );  
```

Comment:
If you need these extra lines for better readability, it's fine; otherwise, use prettier to remove them.